packages/utilities/src/rtl.ts

```diff
@@ -61,3 +60,22 @@ export function getRTLSafeKeyCode(key: number): number {
 
   return key;
 }
+
+/**
+ * Helper utility that gets dir attribute from document elements.
+ * @returns The RTL state of the document (true if in RTL.)
+ */
+export function getDocumentRTL(): boolean {
+  const doc = getDocument();
+  let rtlAttribute;
+
+  if (doc) {
+    rtlAttribute = doc.head.getAttribute('dir');
```

Comment:
i don't think this line will ever be hit. dir is not an attribute you would include on head, as head has no ui content.

Reply:
You're right, this changed the original impl incorrectly. Will fix.